### PR TITLE
feat: sync ALL products (obf, opff, opf)

### PR DESCRIPTION
**Pull request by raphodn**
### What

Improve the product fetch & product sync to also fetch other Flavors

Relies on a branch opened in the `openfoodfacts-python` package : https://github.com/openfoodfacts/openfoodfacts-python/pull/223

### CODE REVIEW

File: app/tasks.py [hunk 1]
```diff
@@ -83,8 +88,8 @@ def import_product_db(db: Session, batch_size: int = 1000) -> None:
     )
     seen_codes = set()
     for product in tqdm.tqdm(dataset):
-        # Skip products without a code
-        if "code" not in product:
+        # Skip products without a code, or with wrong code
+        if ("code" not in product) or (not product["code"].isdigit()):
```
Comment (raphodn) [inline app/tasks.py:92]: some products have strange codes :sweat_smile:

File: app/tasks.py [hunk 2]
```diff
@@ -93,6 +98,7 @@ def import_product_db(db: Session, batch_size: int = 1000) -> None:
             continue
         seen_codes.add(product_code)
 
+        product_lang = product.get("lang", product.get("lc", "en"))
```
Comment (raphodn) [inline app/tasks.py:101]: some products don't have the `lang` key

Comment (raphael0202) [reply app/tasks.py:101]: Can you add it as a comment in the code?

File: pyproject.toml [hunk 3]
```diff
@@ -49,7 +49,7 @@ alembic = "~1.12.1"
 Babel = "~2.13.1"
 fastapi = "~0.103.1"
 jinja2 = "~3.1.3"
-openfoodfacts = "~0.2.0"
```
Comment (raphael0202) [inline pyproject.toml:52]: We will release a new version of Open Food Facts python SDK to have a clean version to pin

**Review by raphael0202 — APPROVED** [review]
Good for me :)
just a couple of comments to address (especially the SDK version one) before merging

File: app/scheduler.py [hunk 4]
```diff
@@ -11,7 +12,10 @@
 
 def import_product_db_job() -> None:
     db = session()
-    import_product_db(db=db)
+    import_product_db(db=db, flavor=Flavor.off)  # Open Food Facts
```
[the remark this answers is not in the document]
Comment (raphodn) [reply app/scheduler.py:15]: yes indeed.

[the remark this answers is not in the document]
Comment (raphodn) [reply app/scheduler.py:15]: can you confirm that we should ignore the off-pro Flavor ?

[the remark this answers is not in the document]
Comment (raphodn) [reply app/scheduler.py]: loop done here : https://github.com/openfoodfacts/open-prices/pull/264/commits/49d9ef1269ffd78882e805b6f1ee5033a0b91c73

[the remark this answers is not in the document]
Comment (raphael0202) [reply app/scheduler.py]: Yes we should ignore them, as they're imported on the public platform.

**Pull request merged**